website/content/docs/operations/stateful-workloads.mdx

```diff
@@ -0,0 +1,246 @@
+and resilience, and can tolerate multiple hardware failures before risking data
+loss. But, it relies on network performance, can have higher latency, be
+lower-throughput, and need more maintenance.
+
+Consider whether Nomad is running in the public cloud or on-premises, and what
+storage options are available in that environment. From there, the most optimal
+choice will depend your organizational and application needs.
+
+### Public cloud
+
+Public cloud providers offer different storage services with various tradeoffs.
+Usually they're comprised of local disks (physically attached and very
+performant, but not very persistent), network attached block devices (performant
+and very persistent, but usually only attachable to one instance at a time), and
+networked shared storage (less performant, but can be accessed from multiple
+instances). 
+
+### AWS 
+
+| AWS service                                                                                  | Availability                                                      | Persistence                                                                      | Performance                                                                                                                                        | Suitability                                                                                                                  |
+|----------------------------------------------------------------------------------------------|-------------------------------------------------------------------|----------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------|
+| [Instance Storage](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/InstanceStorage.html) | Locally on some instance types                                    | Limited, not persistent across instance stops/terminations or hardware failures | High throughput and low latency                                                                                                                    | Temporary storage of information that changes frequently, such as buffers, caches, scratch data, and other temporary content |
+| [Elastic Block Store](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AmazonEBS.html)    | Zonal block devices attached to one or more instances                         | Strongly persistent, with an independent lifecycle                               | [Configurable](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/EBSPerformance.html), but higher latency than Instance Store                    | General purpose persistent storage                                                                                           |
```

Comment:
For this entirely table I'd replace "strongly persistent" with simply "persistent" everywhere. We're not making specific performance claims, and it doesn't add anything.

"Independent lifecycle" seems useful to explain either here in the table or above -- the fact that it's lifecycle is independent _of the VM it's attached to_ is the meaningful bit.